src/Troubleshooting/SettingsDebug.php

```diff
@@ -166,7 +177,11 @@ public function getSettingsGlobalData(): array
 								'checkboxIsChecked' => $this->isCheckboxOptionChecked(self::SETTINGS_DEBUG_DEVELOPER_MODE_KEY, self::SETTINGS_DEBUG_DEBUGGING_KEY),
 								'checkboxValue' => self::SETTINGS_DEBUG_DEVELOPER_MODE_KEY,
 								'checkboxAsToggle' => true,
-								'checkboxHelp' => \__('Outputs multiple options in forms. Every listing will have ID prepended to the label.', 'eightshift-forms'),
+								'checkboxHelp' => \__('
+									Outputs multiple developers options in forms. Here is the list of all the outputs:<br/><br/>
+									<ul>
```

Comment:
Is this complete? 😄

Reply:
Ah, ok, though at first it would be field programmatically 😄

Maybe then just change "Here is the list of all the outputs:" to "Available outputs:"? 😄